assets/js/modules/adsense/index.js

```diff
@@ -74,8 +70,8 @@ export const registerModule = ( modules ) => {
 				__( 'Monetize your website', 'google-site-kit' ),
 				__( 'Intelligent, automatic ad placement', 'google-site-kit' ),
 			],
-			checkRequirements: async () => {
-				if ( ! await isAdBlockerActive() ) {
+			checkRequirements: async ( registry ) => {
+				if ( ! registry.select( STORE_NAME ).isAdBlockerActive() ) {
```

Comment:
This needs to continue to use `__experimentalResolveSelect` to be based on the value of the selector after its resolver has completed.

The value returned here is a promise which resolves to the value of the selector once the resolver is done.